cmd/collectors/rest/rest.go

```diff
@@ -372,7 +374,8 @@ func (r *Rest) PollData() (*matrix.Matrix, error) {
 			f := instanceData.Get(metric.name)
 			if f.Exists() {
 				metr.SetName(metric.label)
-				if err = metr.SetValueFloat64(instance, f.Float()); err != nil {
+				floatValue := HandleIfTimeField(f.String())
```

Comment:
as discussed in chat, let's try an explicit solution

Reply:
Implemented counter(type) way.
May be I have submitted latest change while you were reviewing previous version of PR :)